sponsor/index.html

```diff
@@ -102,8 +105,14 @@ <h4>Sponsorship via Invoice</h4>
       treasurer@osgeo.org
     </a> to express your interest.
       </li>
-      <li>For inclusion on the OSGeo <a href="https://www.osgeo.org/sponsors/">sponsorship</a> page also attach your company logo.</li>
-      <li>Example email sponsoring GeoServer:
+      <li>We need some information to properly thank you for your support:
```

Comment:
I'd tweak the first sentence at the beginning of the "Sponsorship via Invoice" to something like:

"Great for companies and public institutions seeking to perform a one-time donation"

Or something like that

Reply:
"public institutions" good phrasing